code/__DEFINES/layers.dm

```diff
@@ -91,66 +91,88 @@
 #define MASSIVE_OBJ_LAYER 11
 #define POINT_LAYER 12
 
-#define EMISSIVE_BLOCKER_PLANE 12
-#define EMISSIVE_BLOCKER_LAYER 12
-#define EMISSIVE_BLOCKER_RENDER_TARGET "*EMISSIVE_BLOCKER_PLANE"
-
-#define CHAT_LAYER 12.0001 // Do not insert layers between these two values
-#define CHAT_LAYER_MAX 12.9999
-
-#define EMISSIVE_PLANE 13
-#define EMISSIVE_LAYER 13
-#define EMISSIVE_RENDER_TARGET "*EMISSIVE_PLANE"
-
-#define EMISSIVE_UNBLOCKABLE_PLANE 14
-#define EMISSIVE_UNBLOCKABLE_LAYER 14
-#define EMISSIVE_UNBLOCKABLE_RENDER_TARGET "*EMISSIVE_UNBLOCKABLE_PLANE"
-
-#define LIGHTING_PLANE 15
-#define LIGHTING_LAYER 15
+#define EMISSIVE_BLOCKER_PLANE						10
+#define EMISSIVE_BLOCKER_RENDER_TARGET				"*EMISSIVE_BLOCKER_PLANE"
+#define TURF_EMISSIVE_BLOCKER_PLANE					11
+#define TURF_EMISSIVE_BLOCKER_RENDER_TARGET			"*EMISSIVE_BLOCKER_PLANE_TURF"
+#define STRUCTURE_EMISSIVE_BLOCKER_PLANE			12
+#define STRUCTURE_EMISSIVE_BLOCKER_RENDER_TARGET	"*EMISSIVE_BLOCKER_PLANE_STRUCTURE"
+#define ITEM_EMISSIVE_BLOCKER_PLANE					13
+#define ITEM_EMISSIVE_BLOCKER_RENDER_TARGET			"*EMISSIVE_BLOCKER_PLANE_ITEM"
+#define MOB_EMISSIVE_BLOCKER_PLANE					14
+#define MOB_EMISSIVE_BLOCKER_RENDER_TARGET			"*EMISSIVE_BLOCKER_PLANE_MOB"
+
+#define EMISSIVE_PLANE						20
+#define EMISSIVE_RENDER_TARGET				"*EMISSIVE_PLANE"
+#define EMISSIVE_TURF_PLANE					21
+#define EMISSIVE_TURF_RENDER_TARGET			"*EMISSIVE_PLANE_TURF"
+#define EMISSIVE_STRUCTURE_PLANE			22
+#define EMISSIVE_STRUCTURE_RENDER_TARGET	"*EMISSIVE_PLANE_STRUCTURE"
+#define EMISSIVE_ITEM_PLANE					23
+#define EMISSIVE_ITEM_RENDER_TARGET			"*EMISSIVE_PLANE_ITEM"
+#define EMISSIVE_MOB_PLANE					24
+#define EMISSIVE_MOB_RENDER_TARGET			"*EMISSIVE_PLANE_MOB"
+#define EMISSIVE_UNBLOCKABLE_PLANE			31
+#define EMISSIVE_UNBLOCKABLE_RENDER_TARGET	"*EMISSIVE_UNBLOCKABLE_PLANE"
```

Comment:
Could you remove the extra tabs/spaces? They don't really add much of value.

Reply:
Sure

Reply:
Done